### PR TITLE
fix: `ExtendedFloatingActionButton` not accessible by screen readers

### PR DESCRIPTION
**Pull request by TheAabedKhan**

### CODE REVIEW

File: app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt [hunk 1]
```diff
@@ -532,7 +538,10 @@ fun OptionsDialog(
                     if (values == null || !values.contains(key)) option.default else values[key]
 
                 @Suppress("UNCHECKED_CAST")
-                OptionItem(option = option as Option<Any>, value = value, setValue = { set(key, it) })
+                OptionItem(
+                    option = option as Option<Any>,
+                    value = value,
+                    setValue = { set(key, it) })
```
Comment (Axelen123) [inline app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt:544]: Add a newline

**Review by Axelen123 — CHANGES_REQUESTED** [review]
Each `ExtendedFloatingActionButton` already has text. Won't this result in duplicates when the buttons are expanded?

**Comment by optimatrix0** [comment]
Now the button is accessible, I really appreciate that.

File: app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt [hunk 2]
```diff
@@ -532,7 +538,12 @@ fun OptionsDialog(
                     if (values == null || !values.contains(key)) option.default else values[key]
 
                 @Suppress("UNCHECKED_CAST")
-                OptionItem(option = option as Option<Any>, value = value, setValue = { set(key, it) })
+                OptionItem(
+                    option = option as Option<Any>,
+                    value = value,
+                    setValue = {
+                        set(key, it)
+                    })
```
Comment (Axelen123) [inline app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt:546]: Expand it so it looks like this:
```kt
                OptionItem(
                    option = option as Option<Any>,
                    value = value,
                    setValue = {
                        set(key, it)
                    }
		)
```

**Review by Axelen123 — APPROVED** [review]

**Pull request merged**